core/src/com/unciv/ui/tilegroups/TileGroupIcons.kt

```diff
@@ -116,6 +115,29 @@ class TileGroupIcons(val tileGroup: TileGroup){
         }
     }
 
+    // JN updating display of tile yields
+    private fun updateYieldIcon(showTileYields: Boolean) {
+
+        // Hiding yield icons (in order to update)
+        tileGroup.tileYieldGroup.isVisible = false
+
+
+        if (showTileYields) {
+            // Setting up YieldGroup Icon
+            tileGroup.tileYieldGroup.setStats(tileGroup.tileInfo.getTileStats(CivilizationInfo()))
+            tileGroup.tileYieldGroup.setOrigin(Align.center)
+            tileGroup.tileYieldGroup.setScale(0.7f)
```

Comment:
If you use setScale, the framerate will be considerably slower because it needs to compute the transformation on every single tileGroup, and there are a lot of them...
Also, the scaling won't work since the miscLayerGroup has transform = false, no?